pkgs/development/compilers/flutter/default.nix

```diff
@@ -10,25 +12,28 @@ in {
   stable = mkFlutter rec {
     pname = "flutter";
     channel = "stable";
-    version = "1.17.5";
+    version = "1.22.0";
     filename = "flutter_linux_${version}-${channel}.tar.xz";
-    sha256Hash = "0kapja3nh7dfhjbn2np02wghijrjnpzsv4hz10fj54hs8hdx19di";
+    sha256Hash = "0ryrx458ss8ryhmspcfrhjvad2pl46bxh1qk5vzwzhxiqdc79vm8";
     patches = getPatches ./patches/stable;
+    dart = dart_stable;
   };
   beta = mkFlutter rec {
-    pname = "flutter-beta";
-    channel = "beta";
-    version = "1.20.0-7.2.pre";
-    filename = "flutter_linux_${version}-${channel}.tar.xz";
-    sha256Hash = "0w89ig5vi4spa95mf08r4vvwni7bzzdlyhvr9sy1a35qmf7j9s6f";
-    patches = getPatches ./patches/beta;
+    pname = "flutter";
+    channel = "stable";
+    version = "1.22.0";
```

Comment:
Shouldn't it be `beta` instead of `stable`? Latest `beta` version is `1.22.0-12.3.pre`

Reply:
Sorry, For some reason when I run `flutter --version` from beta channel it returns exactly the same for `stable`. But bu the git version of flutter repo, I think you are right, done!

Reply:
Could you please take a look again @babariviere ?